solc/CommandLineInterface.cpp

```diff
@@ -130,20 +131,20 @@ static bool needsHumanTargetedStdout(CommandLineOptions const& _options)
 {
 	if (_options.compiler.estimateGas)
 		return true;
-	if (!_options.output.dir.empty())
+	if (!_options.output.dir.has_value())
 		return false;
-	return
-		_options.compiler.outputs.abi ||
-		_options.compiler.outputs.asm_ ||
-		_options.compiler.outputs.asmJson ||
-		_options.compiler.outputs.binary ||
-		_options.compiler.outputs.binaryRuntime ||
-		_options.compiler.outputs.metadata ||
-		_options.compiler.outputs.natspecUser ||
-		_options.compiler.outputs.natspecDev ||
-		_options.compiler.outputs.opcodes ||
-		_options.compiler.outputs.signatureHashes ||
-		_options.compiler.outputs.storageLayout;
+
+	return _options.compiler.outputs->abi
+		|| _options.compiler.outputs->asm_
+		|| _options.compiler.outputs->asmJson
+		|| _options.compiler.outputs->binary
```

Comment:
Please try to avoid unnecessary reformatting of existing code. This is the style we use:
```suggestion
	return _options.compiler.outputs->abi ||
		_options.compiler.outputs->asm_ ||
		_options.compiler.outputs->asmJson ||
		_options.compiler.outputs->binary ||
```